server/handlers_v2.go

```diff
@@ -297,9 +326,15 @@ func getRecommendationsFillImpacted(
 		}
 	}
 
+	var ruleDisabled bool
 	recommendationList = make([]stypes.RecommendationListView, 0)
 
 	for _, ruleID := range ruleIDList {
+		ruleDisabled = false
+		if _, found := ruleAcksMap[ruleID]; found {
+			ruleDisabled = true
+		}
+
```

Comment:
`_, ruleDisabled := ruleAcksMap[ruleID]` perhaps (ie w/o `if`)

Reply:
of course :) ty